runtime/cudaq/qis/qudit.h

```diff
@@ -10,7 +10,15 @@
 
 #include "execution_manager.h"
 
+using namespace std::complex_literals;
+
 namespace cudaq {
+using complex = std::complex<double>;
+
+namespace ket {
+inline static const std::vector<complex> zero{1. + 0i, 0. + 0i};
```

Comment:
I think it'd be worth it to fill out this type more from the start. Something simple that allows for a variably sized 0 or 1 state. I think it should be added to the python bindings so I could lump it in with my work there